lib/daru/view/adapters/googlecharts/generate_javascript.rb

```diff
@@ -0,0 +1,148 @@
+module GenerateJavascript
```

Comment:
It should be inside `GoogleVisualr ` module, isn't it ?

Reply:
module `GenerateJavascript` is already `include`d in `BaseChart` and `DataTables` class which is in `GoogleVisualr`, so, I think it is not required.